poutyne/framework/experiment.py

```diff
@@ -27,6 +27,139 @@
 
 
 class Experiment:
+    """
+    The Experiment class provides a straightforward experimentation tool for efficient and entirely
+    customizable finetuning of the whole neural network training procedure with PyTorch. The
+    ``Experiment`` object takes care of the training and testing processes while also managing to
+    keep traces of all pertinent information via the automatic logging option.
+
+    Args:
+        directory (str): Path to the experiment's working directory. Will be used for saving
+        model (torch.nn.Module): A PyTorch module.
+        device (torch.torch.device): The device to which the model is sent. If None, the model will be
+            kept on its current device.
+            (Default value = None)
+        logging (bool): Whether or not to log the experiment's progress. If true, various logging
```

Comment:
It should be nice that the documentation of the `train` and `test` methods list what is saved.

Reply:
Agreed. Fixed by b9cde7bbec98cbdb794fc31d0442564e95f60228.